prime-finder.js

```diff
@@ -1,3 +1,23 @@
+'use strict';
+
+//https://codepen.io/KeithB/pen/jwadG  closest solution I could find
 module.exports = function(n) {
-  // do work here
+  var primes = [2];
+  var x = 3;
+  var count = 0;
+  do{
+    for(var y = 2; y < x; y++){
```

Comment:
I'm not sure if this will work. Try to review what's happening to the x variable within the for loop